system_metrics_collector/test/system_metrics_collector/test_linux_memory_measurement.cpp

```diff
@@ -305,10 +305,10 @@ TEST_F(LinuxMemoryMeasurementTestFixture, TestPublishMessage)
   // check expected received message
   const auto received_message = test_receive_measurements->GetLastReceivedMessage();
 
-  EXPECT_EQ(kTestNodeName, received_message.measurement_source_name);
-  EXPECT_EQ(kTestMetricName, received_message.metrics_source);
+  EXPECT_EQ(std::string(kTestMemoryNodeName), received_message.measurement_source_name);
```

Comment:
Is `std::string()` needed? Seems like it wasn't before?

Reply:
Oh I see it's so that comparison with _char[]_ works as expected. What was the original reason for making these _char[]_  initially?

Reply:
See @zmichaels11 comments above.